Array

flametower opened the pull request.

flametower committed:
switch branch

raihaniqbal24 committed:
change from static array to dynamic array

mkamadeus commented on lib/tubesarray.c:
```diff
@@ -0,0 +1,103 @@
+
```
Filenamenya euy

mkamadeus replied:
Komennya sederhanain?

mkamadeus reviewed (changes requested):
Biar rapi aja wkwkw

flametower committed:
implement search on type and owner, major changes
change tulisIsiTab to print every building on field
change array type into dynamic array of pointers

flametower committed:
change filename

flametower commented:
ready to merge
